Data Structures/Array Implementation/ArrayImplementation(Insert,Delete,Linear Search).cpp

```diff
@@ -0,0 +1,94 @@
+//A programme for basic Array Implmentation - Insertion, Deletion, Search (Linear).
+
+#include <stdio.h>
+
+int linearsearch(int value, int n, int array[]) {
+//Function which performs a linear search and returns the index.
+    int p = -1;
+    for (int i = 0; i < n; i++) {
+      if (value == array[i]) {
+        p = i; //Value found in the array
+        break;
+      }
+    }
+    return p;
+}
+void insertvalue(int value,int position,int array[],int n){
+	//A function to insert 
+	for (int p = n - 1; p >= position - 1; p--) {
```

Comment:
Indentation fix required. We won't tell you again and again. Please make sure you follow the Coding Standards. Please strict to either 2 spaces or 4 spaces but don't mix both the styles.

Reply:
Also, write comments to make it clear how this loop is shifting the values to the right.